grub.d/default/xx_mender_font_and_theme_handling.cfg

```diff
@@ -29,8 +30,8 @@ if [ -n "$GRUB_FONT" ]; then
 fi
 
 if [ -n "$GRUB_THEME" ]; then
-    if [ "$(${grub_probe} --target=device "$GRUB_THEME")" = "${mender_kernel_root_base}${mender_rootfsa_part}" ] || \
-        [ "$(${grub_probe} --target=device "$GRUB_THEME")" = "${mender_kernel_root_base}${mender_rootfsb_part}" ]; then
+    if [ "$(${grub_probe} --target=device "$GRUB_THEME")" = "${mender_full_rootfsa_part}" ] || \
```

Comment:
Not related to PR, but what is the `xx` prefix for the filename?

Reply:
The scripts are executed in order, and we really need this one to execute last. I don't remember why I picked `xx` instead of `zz`, but probably it doesn't matter since most of them are numbered.